### PR TITLE
mobile

### PR DESCRIPTION
**Pull request by RyndinaOlena**
Снимок экрана 2024-08-22 в 13 58 37

### CODE REVIEW

File: app/_components/header.tsx [hunk 1]
```diff
@@ -10,49 +10,50 @@ import styles from "../_styles/scss/Header.module.scss";
 import clsx from "clsx";
 import Modal from "./modal";
 import { IoIosArrowDown } from "react-icons/io";
-import Button from '@mui/material/Button';
-import ModalComponent from '../_components/modal'; // Проверьте путь к вашему компоненту
+import Button from "@mui/material/Button";
+import ModalComponent from "../_components/modal"; // Проверьте путь к вашему компоненту
```
Comment (coffpiler) [inline app/_components/header.tsx]: please remove comments in cyrrilic

File: app/_components/header.tsx [hunk 2]
```diff
@@ -10,49 +10,50 @@ import styles from "../_styles/scss/Header.module.scss";
...
-      </div>
-      <div className={styles.headerAccount}>
-        <Link className={scss.btnLogin} href="/">
-          УВІЙТИ
-        </Link>
+    <div>
+      <div className={styles.headerContainer}>
+        <div className={styles.headerLogoWrapper}>
+          <Link className={styles.headerLogo} href={"/"}>
+            <Image src={`/logo.png`} alt="logo" width={55} height={55} />
```
Comment (coffpiler) [inline app/_components/header.tsx:28]: no need to use ` - if no variables in the string 
please replace with ""
```suggestion 
            <Image src="/logo.png" alt="logo" width={55} height={55} />

File: app/_styles/scss/Footer.module.scss [hunk 3]
```diff
@@ -13,57 +13,159 @@
...
       display: flex;
       margin-top: 90px;
       justify-content: space-between;
 
+      @media (max-width: 600px) {
+        flex-direction: column;
+        margin-top: 30px;
+      }
+
+      .navigation_title_non {
```
Comment (coffpiler) [inline app/_styles/scss/Footer.module.scss:96]: please use one styles for classNames 
or snake_case , or camelCase 
please don't mix up and fix in all scss files and tsx files

File: app/_styles/scss/Footer.module.scss [hunk 4]
```diff
@@ -13,57 +13,159 @@
...
+        color: rgba(87, 109, 14, 1);
+        font-size: 18px;
+        font-weight: 700;
+        line-height: 24.5px;
```
Comment (coffpiler) [inline app/_styles/scss/Footer.module.scss:100]: I suggest not use non integer line-heigh 
please replace with 24px

File: app/_styles/scss/Footer.module.scss [hunk 5]
```diff
@@ -13,57 +13,159 @@
...
+        text-transform: uppercase;
+        margin-bottom: 22px;
+
+        @media (max-width: 1100px) {
```
Comment (coffpiler) [inline app/_styles/scss/Footer.module.scss:104]: please store break points into variable and reuse 
@desktop-width: 1100px;
@mobile-width: 600px;

fix in all scss files

File: app/_components/modal.tsx [hunk 6]
```diff
@@ -2,50 +2,42 @@
...
-            </div>
-        </div>
-
-
-    );
+  const { handleClose } = props;
+  const pathname = usePathname();
+  if (!pathname) {
+    <NotFound />;
+  }
```
Comment (coffpiler) [inline app/_components/modal.tsx:27]: ????

Comment (coffpiler) [inline app/_components/modal.tsx:26]: для чого це?

**Comment by RyndinaOlena** [comment]
change class

**Pull request merged**